src/systemTest/java/com/flightstats/hub/resilient/testsuite/HelmReleaseTest.java

```diff
@@ -0,0 +1,38 @@
+package com.flightstats.hub.resilient.testsuite;
+
+import com.flightstats.hub.resilient.BaseTest;
+import com.flightstats.hub.resilient.helm.ReleaseDelete;
+import com.flightstats.hub.resilient.helm.ReleaseInstall;
+import com.flightstats.hub.resilient.testcase.WebhookLifecycleTest;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+@Slf4j
+public class HelmReleaseTest extends BaseTest {
+
+    @Inject
+    @Named("helm.release.name")
+    private String releaseName;
+
+    @Inject
+    @Named("helm.chart.path")
+    private String chartPath;
+
+    @Inject
+    private ReleaseInstall releaseInstall;
+
+    @Inject
+    private ReleaseDelete releaseDelete;
+
+    @Test
+    public void test() {
+        this.releaseInstall.install(this.releaseName, this.chartPath);
+        new JUnitCore().run(WebhookLifecycleTest.class);
```

Comment:
...anyway, I think we might be able to reconfigure this a bit so that we're not manually running tests inside another test. We don't necessarily need to get fancy with a custom test runner, but it might be worth looking at.

Reply:
@BeforeAll and @AfterAll are setup and tear down methods per class. It is run once before all tests in a class. Currently there is no way of doing setup and teardown for the whole test suite in Junit 5. Yes I am thinking of running multiple testcases. More like this. 1) Run a test. 2) Simulate fault in the cluster. 3) Rerun the test to check if all tests pass.